node/beacon_manager.go

```diff
@@ -19,22 +18,23 @@ var _ router.Router = (*beaconManager)(nil)
 type beaconManager struct {
 	router.Router
 	timer         *timer.Timer
-	beacons       validators.Set
+	subnetID      ids.ID
+	beacons       validators.Manager
 	requiredConns int64
 	numConns      int64
 }
 
 func (b *beaconManager) Connected(nodeID ids.NodeID, nodeVersion *version.Application, subnetID ids.ID) {
-	if constants.PrimaryNetworkID == subnetID &&
-		b.beacons.Contains(nodeID) &&
+	if b.subnetID == subnetID &&
```

Comment:
Same comment at the `insecureValidatorManager` - this pattern only works for the primaryNetworkID

Reply:
done